.github/ISSUE_TEMPLATE/docker-compose_test_plan_template.md

```diff
@@ -214,15 +214,42 @@ COMPOSE_FILE=docker-compose.yml:storage/decomposeds3.yml:traefik/opencloud.yml:t
 - `INSECURE=true`
 
 **Test Steps:**
-1. Deploy OpenCloud with S3 storage backend
-2. Verify MinIO container starts
-3. Verify OpenCloud connects to MinIO
-4. Login to OpenCloud
-5. Upload a test file
-6. Verify file is stored in S3 bucket
-7. Download the file
-8. Delete the file
-9. Verify file is removed from S3
+1. Start OpenCloud
+```bash
+   docker-compose up -d
+```
+2. Identify Docker network
+```bash
+   docker network ls
+```
+3. Start MinIO in the same network
+```bash
+   docker run -d \
+    --name minio \
+    --network opencloud-compose_opencloud-net \
+    -p 9000:9000 \
+    -p 9001:9001 \
+    -v minio-data:/data \
+    -e MINIO_ROOT_USER=opencloud \
+    -e MINIO_ROOT_PASSWORD=opencloud-secret-key \
+    --privileged \
+    --user "root" \
```

Comment:
I got `Permission denied` if I don't use `--user` or `Please ensure MinIO binary has write permissions for the backend ` if I use `--user "1000:1000"`

not happy about root user but for testing purpose is fine